launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
How? Do you place the cylinder on top of the sensor or grasp it with the hand? 
If it is the former ok, if it is the latter it is not, since you would need additional packages.